apps/tx-builder/src/pages/CreateTransactions.tsx

```diff
@@ -110,9 +110,9 @@ const CreateTransactions = () => {
 
 export default CreateTransactions
 
-const TransactionsSectionWrapper = styled(Grid)`
+const TransactionsSectionWrapper = styled(Grid)<{ sticky?: boolean }>`
```

Comment:
This `sticky` prop doesn't seem to be used.

Reply:
oh true, nice catch.